### ISSUE TITLE
ClientConnectionManager INFO level logs flooding the Application's log file using JacORB As Dependency.

### ISSUE THREAD

**Issue by Animesh8349** [JacORB/JacORB#240]
The following log message is flooding the Application's log file.
`org.jacorb.orb.giop [org.jacorb.orb.giop.ClientConnectionManager:getConnection:148] - ClientConnectionManager: found ClientGIOPConnection to 10.135.46.193:0
`
The log level for this logger can be turned to DEBUG to prevent it from being logged repeatedly into any application's log file using JacORB as dependency. 

The change suggested is as follows : 
```
**Current Code :** 
...
--
else {
   if( logger.isInfoEnabled()) {
      logger.info("ClientConnectionManager: found " +  clientConnection.getGIOPConnection().toString());
   }
}
...
```

```
**Change Proposed :** 
...
--
else {
   if( logger.isDebugEnabled()) {
      logger.debug("ClientConnectionManager: found " + clientConnection.getGIOPConnection().toString());
   }
}
...
```

**Comment by rnc** [comment]
I'm not against changing that, although shouldn't the creation then be also moved to debug level? (and as an aside, they should both use SLF4J parameterized logger messages). 
PR would be welcome.

**Comment by rnc** [comment]
@Animesh8349 Can you provide a PR (ideally with a test case) please?

**Comment by Animesh8349** [comment]
Hello @rnc ,
Sorry for checking your reply after a long time as I was busy with other tasks.

I have forked the Repo and raised a pull request from the forked repo to the master repo.
Please check if the change can be merged into master ?
The Pull Request Link :  https://github.com/JacORB/JacORB/pull/261

I was not able to add any reviewers as I wasn't getting any option/button for the same over my pull request.
As requested in earlier comment, I am afraid, I'll not be able to provide a working test case as I don't understand the Jacorb source code well enough. I am using it as a project dependency only in order to leverage its APIs.

-Animesh

**Comment by rnc** [comment]
Closed with #261
